src/dbt_client/dbtCoreIntegration.ts

```diff
@@ -909,6 +909,40 @@ export class DBTCoreProjectIntegration
     }
   }
 
+  findPackageVersion(packageName: string) {
+    if (!this.packagesInstallPath) {
+      throw new Error("Missing packages install path");
+    }
+    if (!packageName) {
+      throw new Error("Invalid package name");
+    }
+
+    const dbtProjectYmlFilePath = path.join(
+      this.packagesInstallPath,
+      packageName,
+      "dbt_project.yml",
+    );
+    if (!existsSync(dbtProjectYmlFilePath)) {
+      throw new Error("Package not installed");
+    }
+    const fileContents = readFileSync(dbtProjectYmlFilePath, {
+      encoding: "utf-8",
+    });
+    if (!fileContents) {
+      throw new Error(`${packageName} has empty dbt_project.yml`);
+    }
+    const parsedConfig = parse(fileContents, {
+      strict: false,
+      uniqueKeys: false,
+      maxAliasCount: -1,
+    });
+    if (!parsedConfig?.version) {
+      throw new Error(`Missing version in ${packageName}/dbt_project.yml`);
```

Comment:
maybe use full path, otherwise we need to guess which project in a multi project setup.